script/rubocop-autocorrect.sh

```diff
@@ -25,18 +36,38 @@ git commit --all --message "Regenerate Rubocop's TODO file"
 # mode to find the next failing cop.
 grep "This cop supports safe autocorrection" -A 5 .rubocop_todo.yml\
   | grep '^[A-Z]'\
-  | head "${@:1}"\
   | tr -d :\
-  | while read cop; do
+  | while read cop && [ "$files_per_branch" -gt 0 ]; do
       echo "Trying to autocorrect safely: $cop"
       bundle exec rubocop --regenerate-todo --except "$cop"
 
       echo "Safely autocorrect $cop" > .git/COMMIT_EDITMSG
       echo "" >> .git/COMMIT_EDITMSG
       bundle exec rubocop --autocorrect >> .git/COMMIT_EDITMSG
       if grep -q "offenses corrected" .git/COMMIT_EDITMSG; then
-        git add --all
+
+        unstaged_files=$(git status -s -uno | wc -l)
+        remaining_files=$((unstaged_files - files_per_branch))
+
+        # <dev/tty is needed since stdin is pointing at the file
+        if ((unstaged_files > files_per_branch)) && get_user_confirmation $remaining_files $cop </dev/tty; then
+          git add .
+        else
+          # Commit up to the limit of files defined in the parameter
+          echo "hwewlwkj"
```

Comment:
Looks like a debug echo that was left behind ?

Reply:
lol, whoops.